app/models/user.rb

```diff
@@ -63,19 +63,22 @@ class User < ApplicationRecord
 
   has_many :metadata_description_versions, :dependent => :destroy
 
-
   scope :owners,           -> { where(owner: true) }
   scope :trial_owners,     -> { owners.where(account_type: 'Trial') }
   scope :findaproject_owners, -> { owners.where.not(account_type: [nil, 'Trial', 'Staff']) }
   scope :paid_owners,      -> { non_trial_owners.where('paid_date > ?', Time.now) }
   scope :expired_owners,   -> { non_trial_owners.where('paid_date <= ?', Time.now) }
   scope :active_mailers,   -> { where(activity_email: true)}
 
-  validates :login, presence: true, uniqueness: { case_sensitive: false }, format: { with: /\A[^<>]*\z/, message: "Invalid characters in username"}, exclusion: { in: %w(transcribe translate work collection deed), message: "Username is invalid"}
+  validates :login, presence: true,
+                    uniqueness: { case_sensitive: false },
+                    format: { with: /\A[^<> ]*\z/, message: I18n.t('devise.errors.messages.login.format') },
```

Comment:
Based on comments in Issue, it seems that adding " " spaces in username is not allowed (I assume the validation is in devise gem itself). But adding spaces on username does not trigger the format error on validation, thus I included in regex to improve on the error handling and message.

Reply:
I don't think the i18n reference here is working (locally for me; it does seem to be for Will).  Here's pry in my environment:
```
[12] pry(#<ActiveModel::Errors>):2> messages
=> {
       :email => ["can't be blank"],
    :password => ["can't be blank"],
       :login => ["Translation missing: en.devise.errors.messages.login.format"]
}
```